packages/core/src/utils/SIWXUtil.ts

```diff
@@ -15,57 +15,93 @@ export interface SIWXConfig {
  * @experimental - This is an experimental feature and it is not production ready
  */
 export interface SIWXSession {
-  message: SIWXMessage
+  data: SIWXMessage.Data
+  message: string
   signature: string
+  cacao?: Cacao
 }
 
 /**
  * @experimental - This is an experimental feature and it is not production ready
  */
-export interface SIWXMessage
-  extends SIWXMessageInput,
-    SIWXMessageMetadata,
-    SIWXMessageIdentifier,
-    SIWXMessageMethods {}
+export interface SIWXMessage extends SIWXMessage.Data, SIWXMessage.Methods {}
```

Comment:
This is a bit confusing imo, how would someone differentiate between namespace and interface? 
Maybe `ISIWXMessage` is better?